android/app/src/main/java/com/zulipmobile/notifications/NotificationsPackage.kt

```diff
@@ -0,0 +1,19 @@
+package com.zulipmobile.notifications
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.uimanager.ViewManager
+import java.util.ArrayList
+
+class NotificationsPackage : ReactPackage {
+    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
+        return emptyList()
+    }
+
+    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+        val modules: MutableList<NativeModule> = ArrayList()
+        modules.add(NotificationsModule(reactContext))
+        return modules
```

Comment:
Should be able to write this more simply in Kotlin. Something like (untested):
```suggestion
        return listOf(NotificationsModule(reactContext))
```